module/VuFind/src/VuFind/Connection/Relais.php

```diff
@@ -87,8 +87,8 @@ protected function getDefaultData()
     /**
      * Format the parameters needed to look up an OCLC number in the API.
      *
-     * @param string $oclc   OCLC number to look up
-     * @param string $patron Patron ID (null to use default from config)
+     * @param string      $oclc   OCLC number to look up
+     * @param string|null $patron Patron ID (null to use default from config)
```

Comment:
Would `?string` be a more concise way of representing this?

Reply:
I agree, fixed